### PR TITLE
DOCS-8201 Add Datadog Costs Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Add private beta documentation for Datadog Costs in Cloud Cost Management.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- link: "/cloud_cost_management/custom"
+  tag: "Documentation"
+  text: "Gain insights into your custom costs"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Cloud Cost Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+{{< beta-callout url="#" btn_hidden="true" >}}
```
Comment (alai97) [inline content/en/cloud_cost_management/datadog_costs/_index.md:29]: Google Form link to be added.

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Daily Datadog costs in Cloud Cost Management are in private beta. To request access, complete the form.
+{{< /beta-callout >}}
+
+## Overview
+
+You can enable daily Datadog costs in [Cloud Cost Management][1], giving you visibility into daily Datadog spending across dashboards, notebooks, [cost monitors][2], and Cloud Cost Analytics, along with your additional cloud provider and [SaaS costs][3] for your entire organization.
```
Comment (michaelcretzman) [inline content/en/cloud_cost_management/datadog_costs/_index.md:35]:
```suggestion
Daily Datadog costs gives you visibility into daily Datadog spending across dashboards, notebooks, [cost monitors][2], and Cloud Cost Analytics, along with your entire organization's cloud provider and [SaaS costs][3].  You can enable daily Datadog costs in [Cloud Cost Management][1].
```

Comment (michaelcretzman) [reply content/en/cloud_cost_management/datadog_costs/_index.md:35]: just a suggestion

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+<div class="alert alert-danger">Only existing Cloud Cost Management customers within the parent organization can access daily Datadog costs outside of <a href="https://app.datadoghq.com/billing/usage">Plan and Usage</a>.</div>
+
+[Datadog costs][4] are available for users with [Datadog Admin][5] permissions or custom [Billing Read (`billing_read`) and Usage Read (`usage_read`)][6] permissions on the [**Usage & Cost** page][7].
+
+## Visualize costs
+
+<div class="alert alert-info">Costs outside of <a href="https://app.datadoghq.com/billing/usage">Plan and Usage</a> are based on usage only and do not include historical credits and adjustments.</div>
+
+{{< img src="cloud_cost/datadog_costs/dashboard.png" alt="Datadog costs as an option for the Cloud Cost data source in a dashboard" style="width:100%;" >}}
```
Comment (michaelcretzman) [inline content/en/cloud_cost_management/datadog_costs/_index.md:47]: not sure what this image is showing me relative to the paragraphs before or after it.

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+## Break down costs
+
+You can use out-of-the-box tags to break down and allocate your costs. Use the following tags to filter and group your Datadog cost data.
+
+| Tag Name | Tag Description |
+|---|---|
+| organization | The name of the master or sub-organization. |
+| dimension_name / dimension_id | `dimension_name` is the name of the individual product being billed (for example, `Indexed Logs (15 Day Retention)`). </br></br> `dimension_id` is the snake case version of the product name which is optimized for programmatic use and easier search (for example, `logs_indexed_15day`). |
+| datadog_product / product_name | `datadog_product` is the high-level grouping name for Datadog products (for example, `indexed_logs`). </br></br> `product_name` is the snake case version of the product grouping name which is optimized for programmatic use and easier search (for example, `indexed_logs`). |
+| `<Usage Attribution tags>` | You can add up to three tag keys that are configured in [Usage Attribution][8] with the associated values (for example, `team` or `service`). |
```
Comment (michaelcretzman) [inline content/en/cloud_cost_management/datadog_costs/_index.md:60]:
```suggestion
| `<Usage Attribution tags>` | Tag keys are configured in [Usage Attribution][8] with their associated values (for example, `team` or `service`). You can add up to three tag keys. |
```

Comment (michaelcretzman) [reply content/en/cloud_cost_management/datadog_costs/_index.md:60]: > You can add up to three tag keys

Is this 3 per associated value? I'm assuming this isn't 3 total.

File: content/en/cloud_cost_management/_index.md [hunk 5]
```diff
@@ -203,23 +209,40 @@ Cloud Cost Management collects tags for the AWS, Azure, and Google Cloud integra
...
-Visualize infrastructure spend alongside related utilization metrics to spot potential inefficiencies and savings opportunities:
+## Create a cost monitor
 
-- Add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source.
-  
-{{< img src="cloud_cost/cloud_cost_data_source.png" alt="Cloud Cost available as a data source in dashboard widget creation" style="width:60%;" >}}
+Proactively manage and optimize your cloud spending by creating a [Cloud Cost Monitor][3]. You can choose **Cost Changes** or **Cost Threshold** to monitor your cloud expenses.
 
-- Export a timeseries graph of costs data using the [Metrics API][1].
+{{< img src="cloud_cost/monitor.png" alt="Create a Cloud Cost monitor that alerts on changes in your cost" style="width:100%;" >}}
```
Comment (michaelcretzman) [inline content/en/cloud_cost_management/_index.md:234]:
```suggestion
{{< img src="cloud_cost/monitor.png" alt="Create a Cloud Cost monitor that alerts on cost changes" style="width:100%;" >}}
```

File: content/en/cloud_cost_management/_index.md [hunk 6]
```diff
@@ -8,37 +8,43 @@ further_reading:
...
 
 ## Overview
 
-Cloud Cost Management provides insights for engineering and finance teams to see how changes to infrastructure can affect costs. It enables you to understand trends, allocate spend across your organization, and identify inefficiencies.
-Datadog ingests your cloud cost data and transforms it into queryable metrics. If costs rise, you can correlate the change with usage metrics to determine the root cause.
+Cloud Cost Management provides insights for engineering and finance teams to understand how infrastructure changes impact costs, allocate spend across your organization, and identify inefficiencies.
+
+{{< img src="cloud_cost/overview.png" alt="Gain insights into your cloud provider's cost and usage on the Cloud Costs Overview page in Datadog" style="width:100%;" >}}
+
+Datadog ingests your cloud cost data and transforms them into metrics that you can use in a search query on the [**Analytics** page][1]. If costs rise, you can correlate the increase with usage metrics to determine the root cause.
```
Comment (michaelcretzman) [inline content/en/cloud_cost_management/_index.md:43]:
```suggestion
Datadog ingests your cloud cost data and transforms it into metrics you can use in a search query on the [**Analytics** page][1]. If costs rise, you can correlate the increase with usage metrics to determine the root cause.
```

**Review by michaelcretzman — APPROVED** [review]
Approved with the assumption that you'll consider my minor suggestions.

File: content/en/cloud_cost_management/datadog_costs/_index.md (hunk 4 above)
Comment (alai97) [reply content/en/cloud_cost_management/datadog_costs/_index.md:60]: Based on the [Usage Attribution documentation](https://docs.datadoghq.com/account_management/billing/usage_attribution/), I believe you may only be able to add three tags to scope the usage by.

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+## Permissions
+
+<div class="alert alert-danger">Only existing Cloud Cost Management customers within the parent organization can access daily Datadog costs outside of <a href="https://app.datadoghq.com/billing/usage">Plan and Usage</a>.</div>
+
+[Datadog costs][4] are available for users with [Datadog Admin][5] permissions or custom [Billing Read (`billing_read`) and Usage Read (`usage_read`)][6] permissions on the [**Usage & Cost** page][7].
+
+## Visualize costs
+
+<div class="alert alert-info">Costs outside of <a href="https://app.datadoghq.com/billing/usage">Plan and Usage</a> are based on usage only and do not include historical credits and adjustments.</div>
```
Comment (alai97) [inline content/en/cloud_cost_management/datadog_costs/_index.md:45]:
```suggestion
```

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+<div class="alert alert-warning">Cloud Cost Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Daily Datadog costs in Cloud Cost Management are in private beta. To request access, complete the form.
+{{< /beta-callout >}}
+
+## Overview
+
+Daily Datadog costs gives you visibility into daily Datadog spending across dashboards, notebooks, [cost monitors][2], and Cloud Cost Analytics, along with your entire organization's cloud provider and [SaaS costs][3]. You can enable daily Datadog costs in [Cloud Cost Management][1].
```
Comment (alai97) [inline content/en/cloud_cost_management/datadog_costs/_index.md:35]:
```suggestion
Daily Datadog costs gives you visibility into daily Datadog spending across dashboards, notebooks, [cost monitors][2], and Cloud Cost Analytics, along with your entire organization's cloud provider and [SaaS costs][3]. You can enable daily Datadog costs in [Cloud Cost Management][1].

<div class="alert alert-info">Costs outside of <a href="https://app.datadoghq.com/billing/usage">Plan and Usage</a> are based on usage only and do not include historical credits and adjustments.</div>
```

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+<div class="alert alert-danger">Only existing Cloud Cost Management customers within the parent organization can access daily Datadog costs outside of <a href="https://app.datadoghq.com/billing/usage">Plan and Usage</a>.</div>
+
+[Datadog costs][4] are available for users with [Datadog Admin][5] permissions or custom [Billing Read (`billing_read`) and Usage Read (`usage_read`)][6] permissions on the [**Usage & Cost** page][7].
+
+## Visualize costs
+
+
+{{< img src="cloud_cost/datadog_costs/dashboard.png" alt="Datadog costs as an option for the Cloud Cost data source in a dashboard" style="width:100%;" >}}
+
+Datadog cost data is available in dashboards and notebooks under the **Cloud Costs** data source.
```
Comment (alai97) [inline content/en/cloud_cost_management/datadog_costs/_index.md:50]:
```suggestion
Datadog cost data is available in dashboards and notebooks under the **Cloud Costs** data source. You can create dashboards to monitor your daily costs, identify trends, and optimize resource usage. 
```

**Pull request merged**